x-pack/plugins/elastic_assistant/server/ai_assistant_service/index.ts

```diff
@@ -185,16 +180,6 @@ export class AIAssistantService {
         pluginStop$: this.options.pluginStop$,
       });
 
-      // If v2 is enabled, re-install data stream resources for new mappings
-      if (this.v2KnowledgeBaseEnabled) {
-        this.options.logger.debug(`Using V2 Knowledge Base Mappings`);
-        this.knowledgeBaseDataStream = this.createDataStream({
-          resource: 'knowledgeBase',
-          kibanaVersion: this.options.kibanaVersion,
-          fieldMap: knowledgeBaseFieldMapV2,
```

Comment:
Note that the const `knowledgeBaseFieldMapV2` has now been renamed to `knowledgeBaseFieldMap`. Therefore, `createDataStream` is called for the "v2" mappings on L95